src/realm.h

```diff
@@ -3533,7 +3539,11 @@ RLM_API void
 realm_sync_config_set_after_client_reset_handler(realm_sync_config_t*, realm_sync_after_client_reset_func_t,
                                                  realm_userdata_t userdata,
                                                  realm_free_userdata_func_t userdata_free) RLM_API_NOEXCEPT;
-
+RLM_API void realm_sync_config_set_initial_subscription_handler(realm_sync_config_t*,
```

Comment:
nit: `rerun_on_open` should come after `realm_async_open_task_init_subscription_func_t`